zanata-war/src/main/resources/messages_fr.properties

```diff
@@ -377,7 +377,7 @@ jsf.iteration.CopyProjectValidation.label=Copier les param\u00E8tres de validati
 jsf.iteration.CopyProjectValidations.message=Param\u00E8tres de validation copi\u00E9s \u00E0 partir du projet
 jsf.iteration.CopyProjectType.message=Type de projet copi\u00E9 \u00E0 partir du projet
 jsf.iteration.inheriteLanguage.label=H\u00E9riter les langues \u00E0 partir des param\u00E8tres du projet.
-# translation auto-copied from project Publican, version 4, document publican, author G�rard Baylard
+# translation auto-copied from project Publican, version 4, document publican, author Gérard Baylard
```

Comment:
This is still showing up as being different, but the lines look exactly the same. It must be something to do with the encoding. I hope our client is outputting the comment lines with the right encoding. Comment lines that do not have non-ascii characters are showing up as unchanged. This is a bit of a mystery, and I think we had better figure out what is happening so we know whether it can cause any problems.

- [ ] how did you update this file?